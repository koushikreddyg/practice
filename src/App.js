import React from 'react';
import Router from './Router/Router';

class App extends React.Component {
  render() {
    return (
      <div>
      
      <Router/>
      </div>
    );
  }
}

export default App;
