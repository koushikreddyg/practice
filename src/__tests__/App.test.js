import React from 'react';
import App from '../App';
import ReactShallowRender from 'react-test-renderer/shallow';

test('test for app',()=>{
    const renderer=new ReactShallowRender()
    renderer.render(<App/>)
    expect(renderer.getRenderOutput()).toMatchSnapshot()
})