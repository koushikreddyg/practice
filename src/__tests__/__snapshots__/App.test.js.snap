// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for app 1`] = `
<div>
  <Router />
</div>
`;
